docs/articles_en/documentation/openvino-extensibility.rst

```diff
@@ -124,9 +124,9 @@ The ``Identity`` is a custom operation class defined in :doc:`Custom Operation G
          :language: cpp
          :fragment: [add_frontend_extension]
 
-When Python API is used, there is no way to implement a custom OpenVINO operation. Even if custom OpenVINO operation is implemented in C++ and loaded into the runtime by a shared library, there is still no way to add a frontend mapping extension that refers to this custom operation. In this case, use C++ shared library approach to implement both operations semantics and framework mapping.
+The Python API now supports implementation of custom OpenVINO operations, enabling direct integration within Python code. Still python custom operations won't be loaded into the runtime, so use the C++ shared library approach to implement both operation semantics and framework mapping that refers to this custom operation.
 
-Python can still be used to map and decompose operations when only operations from the standard OpenVINO operation set are used.
+Python can still be used to map and decompose operations when operations from the standard OpenVINO operation set and custom operation implemented in Python are used.
```

Comment:
But only via `ConversionExtension` in case of Python, right?

Reply:
yes